### PR TITLE
Vk service 30

### PR DESCRIPTION
**Pull request by AbrosimovVladislav**

### CODE REVIEW

File: src/main/java/com/vkservice/function/GetGroupMembersIds.java [hunk 1]
```diff
@@ -37,7 +37,7 @@
 					? Integer.toString(i * ONE_MASTER_REQUEST_VOLUME + lastCycleNumberToGet)
 					: Integer.toString(i * ONE_MASTER_REQUEST_VOLUME + ONE_MASTER_REQUEST_VOLUME);
 			JsonElement currentStepJson =
-					vkApiGateway.execute(GET_GROUP_MEMBERS_FUNCTION_NAME, new HashMap<String, String>() {{
+					vkApiGateway.execute(PROCEDURE_NAME, new HashMap<String, String>() {{
 						put(GROUP_ID_PARAM_NAME, groupId);
 						put(OFFSET_START_PARAM_NAME, offsetStart);
 						put(OFFSET_FINISH_PARAM_NAME, offsetFinish);
```
Comment (Armalys) [inline src/main/java/com/vkservice/function/GetGroupMembersIds.java]: need to add line break

File: src/main/java/com/vkservice/function/GetGroupMembersIds.java [hunk 2]
```diff
@@ -14,16 +14,16 @@
 public class GetGroupMembersIds {
 	static final int ONE_MASTER_REQUEST_VOLUME = 20_000;
 	static final String GROUP_ID_PARAM_NAME = "groupId";
-	private static final String GET_GROUP_MEMBERS_FUNCTION_NAME = "getGroupMembers";
+	private static final String PROCEDURE_NAME = "receiveGroupMembers";
```
Comment (AbrosimovVladislav) [inline src/main/java/com/vkservice/function/GetGroupMembersIds.java]: dwecfefer

Comment (AbrosimovVladislav) [reply src/main/java/com/vkservice/function/GetGroupMembersIds.java]: freferf

File: src/main/java/com/vkservice/function/GetGroupMembersIds.java [hunk 3]
```diff
@@ -14,16 +14,16 @@
...
 	private static final String OFFSET_START_PARAM_NAME = "offsetStart";
 	private static final String OFFSET_FINISH_PARAM_NAME = "offsetFinish";
 	private static final GetNumberOfGroupMembers getNumberOfGroupMembers = new GetNumberOfGroupMembers();
 
 	public List<String> execute(VkApiGateway vkApiGateway, String groupId) {
-		return this.getGroupMembers(vkApiGateway, groupId);
+		return this.receiveGroupMembers(vkApiGateway, groupId);
```
Comment (AbrosimovVladislav) [inline src/main/java/com/vkservice/function/GetGroupMembersIds.java]: ferfcefer

**Pull request merged**